internal/service/certificatesigningrequest.go

```diff
@@ -254,6 +290,12 @@ func (h *ServiceHandler) ReplaceCertificateSigningRequest(ctx context.Context, n
 		return nil, api.StatusBadRequest("resource name specified in metadata does not match name in path")
 	}
 
+	ok, err := h.verifyCSRParameters(ctx, csr)
```

Comment:
just something to consider what value does the bool provide here that a non nil error could not solve

Reply:
Fair point.